source/FreeRTOS_IP.c

```diff
@@ -1947,7 +1947,7 @@ static eFrameProcessingResult_t prvProcessIPPacket( const IPPacket_t * pxIPPacke
              * entry. */
             if( ucProtocol != ( uint8_t ) ipPROTOCOL_UDP )
             {
-                if( xCheckRequiresARPResolution( pxNetworkBuffer ) == pdTRUE )
+                if( xCheckRequiresResolution( pxNetworkBuffer ) == pdTRUE )
                 {
                     eReturn = eWaitingARPResolution;
```

Comment:
The enum "eWaitingARPResolution" also should be renamed to "eWaitingResolution" to be in synch with the rest of the changes.